src/v/iceberg/BUILD

```diff
@@ -640,3 +640,21 @@ redpanda_cc_library(
     ],
 )
 
+redpanda_cc_library(
```

Comment:
nit: fine leaving this here for now, but once the datalake subdir has been bazelified it may be worth moving this code into the datalake directory. At least right now I'm thinking of v_iceberg as the code for the Iceberg spec, which isn't related to protobuf. Functionally it probably won't make a difference, but it'll at least enforce a separation between the data path and metadata/catalog path.

Reply:
maybe we should move it now if that's where it should be--unless there is something annoying about moving it.

Reply:
will move it right away